chore: add info about cookie methods in v2 upgrade guide (#10055)
supabase/supabase: apps/reference/_supabase_js/upgrade-guide.mdx

</TabItem>
</Tabs>

#### Cookie methods

The cookie related methods like `setAuthCookie` and `getUserByCookie` have been removed.

For Next.js you can use the [Auth Helpers](https://supabase.com/docs/guides/auth/auth-helpers/nextjs) to help you manage cookies.

> If you can't use the Auth Helpers, you can follow [this approach](https://supabase.com/docs/guides/auth/server-side-rendering) for server-side-rendering.

Reviewer: ```suggestion
If you can't use the Auth Helpers, you can follow [this approach](https://supabase.com/docs/guides/auth/server-side-rendering) for server-side rendering.
```

> If you prefer not to use the auth helpers package you can follow [this approach](https://supabase.com/docs/guides/auth/server-side-rendering) for server-side-rendering.

Reviewer: ```suggestion
If you can't use the Auth Helpers, you can follow [this approach](https://supabase.com/docs/guides/auth/server-side-rendering) for server-side-rendering.
```
